transport/kubernetes-jobmonitor/src/main/kotlin/MonitorConfig.kt

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (C) 2024 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.eclipse.apoapsis.ortserver.transport.kubernetes.jobmonitor
+
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.seconds
+
+import org.eclipse.apoapsis.ortserver.config.ConfigManager
+import org.eclipse.apoapsis.ortserver.config.Path
+
+/**
+ * A class collecting the configuration options supported by the Kubernetes Job Monitor component.
+ */
+data class MonitorConfig(
+    /** The name of the Kubernetes namespace this component should monitor. */
+    val namespace: String,
+
+    /** The interval in which the Reaper component should run. */
+    val reaperInterval: Duration,
+
+    /**
+     * The maximum age of a job for being deleted by the Reaper. During a Reaper run, it deletes all completed that
+     * are older than this value.
+     */
+    val reaperMaxAge: Duration,
+
+    /** The interval in which the lost jobs detection should run. */
+    val lostJobsInterval: Duration,
+
+    /**
+     * The minimum age of a job in the database to be taken into account by the lost job finder component. This is
+     * needed to prevent certain race conditions with jobs that have just been added to the database, but are not yet
+     * propagated to Kubernetes.
+     */
+    val lostJobsMinAge: Duration,
+
+    /**
+     * A duration that defines an interval in which a job should be considered as recently processed. The job handler
+     * keeps a list of the jobs that have been deleted during this interval to prevent duplicate processing of jobs.
+     */
+    val recentlyProcessedInterval: Duration,
+
+    /** Flag whether the watching component should be active. */
```

Comment:
In the log message above, it is "the watcher component".
It should be consistent.

(more cases underneath)

Reply:
I have replaced "watching component" by "watcher component" in two places. I do not want to change the names of configuration properties though.